src/main/java/io/github/moulberry/notenoughupdates/util/ApiUtil.java

```diff
@@ -225,6 +226,8 @@ private CompletableFuture<String> requestString0() {
 						// Not in the sense that this will hold in most cases (although that as well),
 						// but in the sense that any violation of this better have a good reason.
 						return IOUtils.toString(inputStream, StandardCharsets.UTF_8);
+					} catch (Throwable t) {
+						throw new RuntimeException(t);
```

Comment:
No point im pretty sure. Those should be caught by the completedfuture monad. If this needs to be done use a CompletionException to avoid double wrapping